packages/bpk-component-modal/index.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { BpkModalV2 } from './src/BpkModalV2/BpkModal';
+import themeAttributes from './src/themeAttributes';
+import BpkModal from './src/BpkModal';
+import type { Props } from './src/BpkModal';
+// @ts-expect-error Untyped import. See `decisions/imports-ts-suppressions.md`.
+import { propTypes, defaultProps } from './src/legacy-prop-types';
```

Comment:
Just checking why are we keeping the legacy props? 

Am I right in assuming this is because there is another component using these right now?

Reply:
It's actually because we export them and they're also used by our examples and didn't want to make this into a breaking change. There's a few components where we export the prop types and default props, but I thought we could remove them once we migrate our examples?